src/System/executable_query.rs

```diff
@@ -12,6 +13,7 @@ use super::save_component::PruningStrategy;
 
 pub enum QueryResult {
     Refinement(bool),
+    Reachability(bool, Vec<String>), //Skal ændre String til Edge
```

Comment:
Please do implement the change, and have comments in english

Reply:
This should be changed to Vec<Edge>, once the Protobuf is ready to be merged, and therefore we can not implemented yet.

Have changed the comment to English :)

Reply:
When will the Protobuf be ready? I'm not sure what I feel about merging something that will be changed soon anyway...

Reply:
Since this change will only implement how to parse the input we have no result to give yet. And we do not yet have the means to create a result, which we could send back via protobuf. We wanted this in main because it also brings the idea of locations, states, and clocks to the parser which other groups might also need. So if we merge this the other groups could make use of the functions which we bring. We will be done with the protobuf at least at the end of the next sprint. But the edge struct is also defined only in protobuf as an idea and not yet fully decided what it will contain. We could make a temp stuct for edges instead, would that be better?

Reply:
@t-lohse friendly reminder =)